docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md

```diff
@@ -122,10 +123,44 @@ Applications rely on the .NET Framework Resource Manager, represented by the <xr
+     * If a satellite still has not been found, the AssemblyLoadContext causes the AppDomain to trigger an <xref:System.AppDomain.AssemblyResolve?displayProperty=nameWithType> event to indicate that it is unable to find the satellite assembly. If you choose to handle the event, your event handler can load and return a reference to the satellite assembly.
+
+2. If a Satellite is found, the runtime then searches the satellite assembly for the requested resource. If it finds the resource in the assembly, it uses it. If it doesn't find the resource, it continues the search.
+
+     > [!NOTE]
+     >  To find a resource within the satellite assembly, the runtime searches for the resource file requested by the <xref:System.Resources.ResourceManager> for the current <xref:System.Globalization.CultureInfo.Name?displayProperty=nameWithType>.  Within the resource file it seaches for the requested resource name.  If either is not found, the resource is treated as not found.
+
+3. The runtime next searches the parent culture assemblies through many potential levels, each time repeating steps 1 & 2.
+
+     The parent culture is defined as an appropriate fallback culture. Consider parents as fallback candidates, because providing any resource is preferable to throwing an exception. This process also allows you to reuse resources. You should include a particular resource at the parent level only if the child culture doesn't need to localize the requested resource. For example, if you supply satellite assemblies for `en` (neutral English), `en-GB` (English as spoken in the United Kingdom), and `en-US` (English as spoken in the United States), the `en` satellite would contain the common terminology, and the `en-GB` and `en-US` satellites could provide overrides for only those terms that differ.
+
+     Each culture has only one parent, which is defined by the <xref:System.Globalization.CultureInfo.Parent%2A?displayProperty=nameWithType> property, but a parent might have its own parent. The search for parent cultures stops when a culture's <xref:System.Globalization.CultureInfo.Parent%2A> property returns <xref:System.Globalization.CultureInfo.InvariantCulture%2A?displayProperty=nameWithType>; for resource fallback, the invariant culture is not considered a parent culture or a culture that can have resources.
+
+4. If the culture that was originally specified and all parents have been searched and the resource is still not found, the resource for the default (fallback) culture is used. Typically, the resources for the default culture are included in the main application assembly. However, you can specify a value of <xref:System.Resources.UltimateResourceFallbackLocation.Satellite> for the <xref:System.Resources.NeutralResourcesLanguageAttribute.Location%2A> property of the <xref:System.Resources.NeutralResourcesLanguageAttribute> attribute to indicate that the ultimate fallback location for resources is a satellite assembly, rather than the main assembly.
```

Comment:
- UltimateResourceFallbackLocation.Satellite should be ?displayProperty.nameWithType
- NeutralResourcesLanguageAttribute.Location property of the ...attribute -- can be shortened to \<xref:System.Resources.NeutralResourcesLanguageAttribute.Location?displayProperty=nameWithType> property
- no comma before "rather"

Reply:
Done